pkgs/by-name/em/emocli/package.nix

```
@@ -0,0 +1,32 @@
+{ lib, nimPackages, musl, fetchurl, fetchFromGitLab }:
+
+nimPackages.buildNimPackage rec {
+  pname = "emocli";
+  version = "1.0.0";
+  src = fetchFromGitLab {
+    owner = "AsbjornOlling";
+    repo = "emocli";
+    rev = "v${version}";
```

Because of how you created the tag this should be:
```suggestion
    rev = version;
```